ui/src/translations/fr.json

```
@@ -185,5 +185,6 @@
   "no_system_partition": "Aucune partition système",
   "error_system_partitions": "Échec de la récupération de la partition système.",
   "show_cluster_avg": "Montrer la moyenne du Cluster",
-  "cluster_avg": "Cluster moy"
+  "cluster_avg": "Cluster moy",
+  "dashboard": "Dashboard"
```

Maybe we should translate this ?

```suggestion
  "dashboard": "Tableau de bord"
```

---

After discussing with @Cuervino,  we decided not to translate the "dashboard" for the following reason.
1. In french, we also use "dashboard" in the professional environment.
2. "Tableau de bord" would be too long as a sidebar item.